TERMINOLOGY.md

```diff
@@ -1,53 +1,70 @@
+
+
+**Randomization.** Unlike [scikit-learn estimators](https://scikit-learn.org/stable/glossary.html#term-estimator), estimators in fairlearn can produce randomized predictors. Randomization of predictions is required to satisfy many definitions of fairness. Because of randomization, it is possible to get different outputs from the predictor's `predict` method on identical data. For each of our methods, we provide explicit access to the probability distribution used for randomization.
+
+## Fairness of AI systems
+
+AI systems can behave unfairly for a variety of reasons. Sometimes it is because of societal biases reflected in the training data and in the decisions made during the development and deployment of these systems. In other cases, AI systems behave unfairly not because of societal biases, but because of characteristics of the data (e.g., too few data points about some group of people) or characteristics of the systems themselves. Because it can be hard to distinguish between these reasons (and these reasons are not mutually exclusive and often exacerbate one another), we define whether an AI system is behaving unfairly in terms of its impact on people – i.e., in terms of harms – and not in terms of specific causes, such as societal biases, or in terms of intent, such as prejudice.
+
+**Usage of the word _bias_.** Since we define fairness in terms of harms rather than specific causes (such as societal biases), we avoid the usage of the words _bias_ or _debiasing_ in describing the functionality of fairlearn.
+
+### Types of harms
+
+There are many types of harms (see, e.g., the [keynote by K. Crawford at NIPS 2017](https://www.youtube.com/watch?v=fMym_BKWQzk)). Fairlearn is most applicable to two kinds of harms:
+
+- _Allocation harms_. These harms can occur when AI systems extend or withhold opportunities, resources, or information. Some of the key applications are in hiring, school admissions, and lending.
+
+- _Quality-of-service harms_. Quality of service refers to whether a system works as well for one person as it does for another, even if no opportunities, resources, or information are extended or withheld.
+
+### Fairness assessment and unfairness mitigation
+
+In fairlearn, we provide tools to assess fairness of predictors for classification and regression. We also provide tools that mitigate unfairness in classification and regression. In both assessment and mitigation scenarios, fairness is quantified using disparity metrics as we describe below.
+
+#### Group fairness, group-membership features
+
+There are many approaches to conceptualizing fairness. In fairlearn, we follow the approach known as group fairness, which asks: _Which groups of individuals are at risk for experiencing harms?_
+
+The relevant groups (also called subpopulations) are defined using **group-membership features**, which represented as a data frame, an array or a vector called `group_membership`. Alternative terminology for group-membership features:
```

Comment:
Are you sure `group_membership` makes sense for the arg? I would be very confused as a user, and I'd think that I have to preprocess my matrix (for multiple attributes) into a vector that represents groups. At least we should do a great job in documenting it if we go with it. Should confirm asap.

Adding @bethz  @riedgar-ms  @vingu to settle this.

Reply:
What about allowing the grouping data to be taken from `X`? The above implies that the `group_membership` argument will always have to be present, even if everything is in `X`

Reply:
Given that currently we only support vectors for group_membership, your conclusion would be correct! But jokes aside--we could expand to something like group_membership_features if that would make things clearer. After talking with a bunch of folks, I'd go for clarity over conciseness.

Reply:
That's orthogonal to my comment. I had thought that we wanted that `group_membership_features` (aka `aux_data` aka....) argument to be optional? We were going to modify the `Moment` constructors so that they could specify whether their data were to be taken from `X` or from this other arg.

Reply:
[I was replying to @romanlutz ]... that's correct. I was going to take a stab at the initial rewrite of Moment once you update GridSearch.

Reply:
see #64

Reply:
resolving to sensitive_features